.github/workflows/test_with_trezor-user-env.yml

```diff
@@ -0,0 +1,84 @@
+name: Test with trezor-user-env
+on:
+  workflow_call:
+    inputs:
+      methods:
+        description: "List of methods to include in tests (example applySettings,applyFlags,getFeatures)"
+        type: "string"
+        required: false
+      test-pattern:
+        description: "Test pattern to use (e.g. `init` or `methods`)"
+        type: "string"
+        required: true
+
+jobs:
+  test-nodejs:
+    name: Test NodeJS (${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+    services:
+      trezor-user-env:
+        image: "ghcr.io/trezor/trezor-user-env:latest"
+        ports:
+        - 9001:9001
+        - 21326:21326
+        - 21325:21326
+        env:
+          SDL_VIDEODRIVER: "dummy"
+          USE_TX_CACHE: true
+          USE_WS_CACHE: true
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: yarn
+        cache-dependency-path: "**/yarn.lock"
+    - run: yarn install
+    # Use `-d` to disable Docker, since it's started under `trezor-user-env`
+    - run: ./tests/run.sh -d -s 'yarn test:integration ${{ inputs.test-pattern }}'
+      env:
+        TESTS_INCLUDED_METHODS: '${{ inputs.methods }}'
+
+  test-karma:
+    name: Test Karma
+    runs-on: ubuntu-18.04
```

Comment:
again, can be latest maybe?

Reply:
I'm unfamiliar with how `karma` and `xvfb-run` works, but I tried upgrading it to `ubuntu-latest` in 86c0e578ca1989a857504638c1c5244f32cf0817 and it seems to work.